torch/lib/c10d/ProcessGroupMPI.hpp

```diff
@@ -87,15 +90,23 @@ class ProcessGroupMPI : public ProcessGroup {
               OpType::UNKNOWN,
               profilingTitle,
               inputTensors),
-          outputTensors_(std::move(outputTensors)) {}
+          outputTensors_(std::move(outputTensors)),
+          future_(c10::make_intrusive<at::ivalue::Future>(
+            c10::ListType::create(c10::TensorType::get()))) {}
```

Comment:
could you please run clang-format on the new code? I think there needs to be two additional spaces here

Reply:
Done. my clang ext in VS broke for some reason, sorry about that